ENIGMAsystem/SHELL/Widget_Systems/widgets_mandatory.h

```diff
@@ -23,7 +23,21 @@
 
 #include <string>
 
-#define DEBUG_MESSAGE(msg, severity) ::enigma_user::show_debug_message((std::string) (msg) + " | " __FILE__ ":" + std::to_string(__LINE__), (severity))
+#ifdef ABORT_ALL_ERRORS
```

Comment:
Instead of duplicating the macro for all permutations of these two variables, I'd ensure ABORT_ON_ALL_ERRORS was defined to false by default and then include it in the if().

Reply:
@JoshDreamland is what i have no ok? for both things you addressed